wagtail/embeds/finders/embedly.py

```diff
@@ -52,7 +52,9 @@ def find_embed(self, url, max_width=None, key=None):
 
         # Convert photos into HTML
         if oembed["type"] == "photo":
-            html = '<img src="%s" alt="">' % (oembed["url"],)
```

Comment:
Question: Should oembed images like this reuse the `default_attrs`?

Reply:
probably - but can be done as a separate PR if needed.